activity_browser/settings.py

```diff
@@ -77,46 +74,55 @@ def __init__(self, filename: str):
         super().__init__(ab_dir, filename)
 
     @staticmethod
-    def update_old_settings(directory: str, filename: str) -> None:
-        """Recycling code to enable backward compatibility: This function is only required for compatibility
-        with the old settings file and can be removed in a future release
+    def _update_old_settings(directory: Path, filename: str) -> None:
+        """Update old settings, backwards compatibility method.
+
+        This function is only required for compatibility with the old settings file
+        and can be removed in a future release.
```

Comment:
Any idea how old the old settings file is? Maybe we can remove this method now?

Reply:
It looks like the change it at least 4 years ago, so it should be fine to remove yes.